### PR TITLE
Add dependency matrix for tpm2-tools

### PR DESCRIPTION
**Pull request by idesai**
Progress issue #4

Signed-off-by: Imran Desai <imran.desai@intel.com>

### CODE REVIEW

**Comment by idesai** [comment]
@williamcroberts @PeterHuewe, once this is merged I will leave a pointer to this location in the tools project and delete it from the tools wiki. Also, I think it might be a good idea to create separate markdowns for other projects like pkcs11, totp, tss-engine, etc. since the dependencies might vary.

**Comment by AndreasFuchsSIT** [comment]
The _posts folder ends up under tutorials.
IMHO we should put this on one of the other pages.

My proposal: Let's rename the "Packaging" to "Versions" or similar and put the dependency-matrix into there as well ?

File: _posts/tpm2-tools-dependency-matrix.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Dependency Version Matrix
```
Comment (williamcroberts) [inline _posts/tpm2-tools-dependency-matrix.md:1]: I thought Andreas said things under _posts are tutorials, don't we want to drop this as the information is in versions.md. I think you want to delete that

File: software.md [hunk 2]
```diff
@@ -49,3 +49,10 @@ permalink: /software/
 - [GNOME Keyring](https://wiki.gnome.org/Projects/GnomeKeyring/SecurityFAQ#I_have_a_TPM_.28Trusted_Platform_Module.29_chip_on_my_machine._Can_I_use_it_to_protect_my_passwords.3F)
 
 Please feel free to also add notes to this list wrt means of integration, e.g. if a project could be enabled using tpm2-pkcs11 and p11-kit because it already provides a pkcs11 interfaces for authentication.
+
+# Packaging
```
Comment (williamcroberts) [inline software.md:53]: @idesai didn't @AndreasFuchsSIT want this as Version?

Comment (idesai) [reply software.md:53]: You are correct. I will move it back.
My reasoning was that the content from repology seemed apt in the document that lists all the software that currently uses tpm2-software as a dependency. But repology does more than that. I actually list the version number that gets packaged.

**Pull request merged**